### PR TITLE
Move down description.txt text

### PR DESCRIPTION
**Pull request by ghost**
#3910

### CODE REVIEW

**Comment by sofar** [comment]
This is a fix for a legitimate bug - add to 0.4.14 milestone?

**Comment by ObaniGemini** [comment]
Too much fixes are coming during the freeze xD

File: builtin/mainmenu/tab_texturepacks.lua [hunk 1]
```diff
@@ -25,7 +26,6 @@ local function filter_texture_pack_list(list)
 	end
 
 	table.sort(retval)
-	table.insert(retval, 1, fgettext("None"))
```
Comment (kilbith) [inline builtin/mainmenu/tab_texturepacks.lua]: Wait a moment... you aren't supposed to make that.

`None` should remain at the top, not sorted alphabetically. Stop messing around with unrelated things.

File: builtin/mainmenu/tab_texturepacks.lua [hunk 2]
```diff
@@ -50,14 +50,16 @@ end
...
 			"textlist[4,0.25;7.5,5.0;TPs;"
 
 	local current_texture_path = core.setting_get("texture_path")
 	local list = filter_texture_pack_list(core.get_dir_list(core.get_texturepath(), true))
 	local index = tonumber(core.setting_get("mainmenu_last_selected_TP"))
 
-	if index == nil then index = 1 end
+	if index == nil then
+		index = 1
+	end
```
Comment (kilbith) [inline builtin/mainmenu/tab_texturepacks.lua]: It wasn't needed to make a newline here...

Also replace it by `if not index then`.

**Comment by Rui914** [comment]
Updated.

**Comment by kwolekr** [comment]
:+1:

**Comment by nerzhul** [comment]
:+1:

**Comment by kwolekr** [comment]
https://github.com/minetest/minetest/commit/c8ff11b417ae2c3a156bcaca819eaeec334c76b5

**Pull request closed without merging**